src/sfeTkArdI2C.h

```diff
@@ -211,6 +237,10 @@ class sfeTkArdI2C : public sfeTkII2C
     TwoWire *_i2cPort;
 
   private:
+sfeTkError_t writeRegisterRegionAddress(uint8_t *devReg, size_t regLength, const uint8_t *data, size_t length);
```

Comment:
formatting -> these shoudl be indented and align with above methods. Run the file through the formatter in your editor.  Details here: https://github.com/sparkfun/.github/blob/main/SFECodeStyleGuide.md

Reply:
Functions have been indented and are now aligned